src/Prover-Axelar.sol

```diff
@@ -225,4 +283,71 @@ contract DealClient is AxelarExecutable {
     ) internal pure returns (string memory) {
         return Strings.toHexString(uint256(uint160(_addr)), 20);
     }
+
+    function asciiBytesToUint(
+        bytes memory asciiBytes
+    ) public pure returns (uint256) {
+        uint256 result = 0;
+        for (uint256 i = 0; i < asciiBytes.length; i++) {
+            uint256 digit = uint256(uint8(asciiBytes[i])) - 48; // Convert ASCII to digit
+            require(digit <= 9, "Invalid ASCII byte");
+            result = result * 10 + digit;
+        }
+        return result;
+    }
+
+    function convertAsciiHexToBytes(
```

Comment:
Did you figure out what part of the system rejected raw bytes? We are paying a 2x size overhead doing things this way + all of the gas overhead of doing the conversion on chain.  Its ok to keep this for now but if we want to move to using this in production its better to fix the software rejecting raw byte labels